src/v/redpanda/admin/server.cc

```diff
@@ -1698,6 +1698,28 @@ void config_multi_property_validation(
         errors[ss::sstring(name)] = ssx::sformat(
           "{} requires schema_registry to be enabled in redpanda.yaml", name);
     }
+
+    // Cloud storage cache size validation
+    {
+        const auto& cloud_storage_cache_size
```

Comment:
These temporaries don't help with readability too much. The names are very similar to the variables below and don't really make it clear what the difference is: `cloud_storage_cache_size` vs `cache_size`.

I think you can probably go right to the value you want:
```
auto cache_size 
    = updated_config.cloud_storage_cache_size();
auto cache_size_pct
    = updated_config.cloud_storage_cache_size_percent().value_or(0.0);
```

Or call the intermediate value `cache_size_property` or something to make it clear that it's the config property and not the raw value.

Reply:
Updated temporaries here. The reason for the temporaries is for use of both the values and names from the properties, but I've made it a little less redundant here.